root/inc/notification.html

```diff
@@ -24,6 +24,17 @@
             text   = "The maintainer of this distribution is looking for someone to take over!",
             body   = "Hello,\r\n\r\nI'm interested in taking over the %s module. If you can grant me the privilege I will release the next version.\r\n\r\n"
         },
+        DEPRECATED = {
+            title  = "Deprecated.",
+            subject = "Take over %s?",
+            contactwhom = "them",
+            text   = "The maintainer of this distribution has indicated that it is deprecated and no longer suitable for use.",
+            body   = "Hello,\r\n\r\nI'm interested in taking over the %s module. If you can grant me the privilege I will release the next version.\r\n\r\n"
```

Comment:
Deprecating a module isn't really an indication of looking for maintainers by itself. There isn't any reason to present a mail option.

Reply:
Marvelous!  That could actually help make this simpler...

By not having to provide an indication that we could be looking for someone to take over the module, the implementation for this could be refactored and simplified further.

Reply:
Shall I proceed ahead with some refactoring here, in the mind of setting this up such that when a Distribution is marked as Deprecated, that we do not provide the statement/option to e-mail the author?  After which point, I can then amend the PR w/new commits.

Reply:
Sounds good!